acceptancetests/repository/charms/appdata-sink/actions/echo

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -eux
+
```

Comment:
Is it worth copying in the actions?

Reply:
I'll ditch them from appdata-sink and update the -source ones to have the unit try to set the token unconditionally.